telosnetwork/telos-wallet
# #602 | Initial allowances page

# Fixes #602

## Description
This PR adds the page and control elements for allowances

## Test scenarios
- `git fetch --all && git checkout 602-allowances-layout && yarn dev`
- go to http://localhost:8081
- log in
    - you should see a new allowances menu item
- go to allowances page
    - it should match the designs (some text has been changed)

## Checklist:
